chain-impl-mockchain/src/lib.rs

```diff
@@ -0,0 +1,5 @@
+extern crate serde;
+#[macro_use]
+extern crate serde_derive;
+
+pub mod mock;
```

Comment:
don't call this module `mock`. We already know that we are using the mockchain.

can you split the module actually ?

* error (with Result/Error defined in there);
* key (PublicKey, PrivateKey, Signature...);
* address (Address);
* transactions (TransactionId, Transaction, SignedTransaction, Witness, UtxoPointer, Output...);
* Block (Block, BlockId, SlockId, ...)

thanks

Reply:
Done